app/controllers/users_controller.rb

```diff
@@ -0,0 +1,23 @@
+class UsersController < ApplicationController
+
+	def new
+		@user = User.new
+	end
+
+	def create
+		@user = User.new(
+			name: params[:user][:name],
+			email: params[:user][:email],
+			password: params[:user][:password],
+			password_confirmation: params[:user][:password_confirmation],
+			department: params[:user][:department],
+			admin: params[:user][:admin]
+			)
+
+		if @user.save
+			redirect_to root_url
+		else
+			render "new"
+		end
+	end
```

Comment:
@nozomi-matsuoka
For this project I want us to use **strong parameters** . I will discuss more on this to the class.
https://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters/jp 


In this file, please provide a private method at the end of the class: 
```
private
  def user_params
    params.require(:user).permit(:name, :email, :password, :password_confirmation, :department, :admin)
  end
```

Then change `line 8-15` to:
`@user = User.new(user_params`)